BEIMA.Backend/MongoService/Device.cs

```diff
@@ -166,31 +211,41 @@ public void SetLocation(ObjectId buildingId, string notes, string latitude, stri
         public void SetLastModified(DateTime? date, string user)
         {
             //Check if null, if they are, then use defualt values
-            date ??= DateTime.Now.ToUniversalTime();
+            var modifiedDate = (date ??= DateTime.Now.ToUniversalTime());
```

Comment:
Why do we need to create a temp variable here?

Reply:
As a side note, I think you can also use DateTime.UtcNow instead of DateTime.Now.ToUniversalTime().

Reply:
It didn't like assigning  DateTime? to a DateTime

Reply:
If you include the ??= it should be fine no?

Reply:
You could try using:
LastModified.Date date = date ?? DateTime.UtcNow;

Reply:
You can use this:

date ??= DateTime.UtcNow;

And then when you actually set the Date property:

LastModified.Date = (DateTime)date;

Reply:
Changed to DateTime.UtcNow. Could have sworn it didn't like the assignment but it works just fine now.